src/app/migrations/password/password.1.angular.txt

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: `
+    <form clrForm>
+      <clr-password-container>
```

Comment:
I have a few questions with `password.1` example when I try to understand it

* Angular demo doesn't provide label but Core demo have - why?
* Core demo doesn't snow the label anyway ? - Bug or bath implementation 

I know that the reason for that is `labelLayout="hidden-label"` But why to make it like that? 

The basic example must show how to create the most basic form of password field and go to properties that could create confusion 

So if the Angular component doesn't have a label there is no need to implement the label and then hide it on the Core example. Because this could be a bad pattern if someone just copy-paste the code. 

I think the basic example here must include, password with a label without `<form>` wrapper and any additional attributes.